src/exml.erl

```diff
@@ -14,20 +14,15 @@
 -include("exml_stream.hrl").
 
 -export([parse/1]).
-
 -export([to_list/1,
          to_binary/1,
          to_iolist/1,
          xml_size/1,
          xml_sort/1,
          to_pretty_iolist/1]).
-
 -export([]).
 
--export_type([attr/0,
-              cdata/0,
-              element/0,
-              item/0]).
+-export_type([attr/0, cdata/0, element/0, item/0]).
```

Comment:
All exported types are flattened onto one line. Expected: one type per line should be preserved.

Reply:
Are you using `inline_items => true` in your options, by any chance?
If not, then this is a bugs and you should report it in the rebar3_format repo.

Reply:
@elbrujohalcon we use the default setting, which as far as I know is `inline_items => {when_over, 25}`

Reply:
The issue is here: AdRoll/rebar3_format#88